assets/js/views/app.js

```diff
@@ -0,0 +1,38 @@
+var AppView = Backbone.View.extend({
+  template: AppTemplates.app,
```

Comment:
Check your template name based on what files are in your `assets/templates` directory.